src/modules/projects/components/tables/ProjectHouseholdsTable.tsx

```diff
@@ -43,14 +45,24 @@ const BASE_COLUMNS: ColumnDef<OneHouseholdClient>[] = [
   CLIENT_COLUMNS.age,
   {
     header: 'Relationship',
-    render: (householdClient) => (
-      <HmisEnum
-        key={householdClient.id}
-        value={householdClient.relationshipToHoH}
-        enumMap={HmisEnums.RelationshipToHoH}
-        whiteSpace='nowrap'
-      />
-    ),
```

Comment:
I think I prefer keeping DNC visible, in this new table design.  I would stay stick with what you had! I was just avoiding making design changes with https://github.com/greenriver/hmis-frontend/pull/1014 since that wasn't the focus.

<img width="521" alt="Screenshot 2025-01-15 at 12 36 45 PM" src="https://github.com/user-attachments/assets/6f1b856b-1f70-4923-90f3-20f584d4a807" />

Reply:
Gotcha, thank you! I reverted this.